src/pages/Thanks/ThanksPage.vue

```diff
@@ -544,6 +559,20 @@ export default {
 				this.enableShortVersion = true;
 			}
 		}
+
+		// Thanks Bagdes Experiment
+		if (this.optedIn) {
+			const { version } = trackExperimentVersion(
+				this.apollo,
+				this.$kvTrackEvent,
+				'thanks',
+				THANKS_BADGES_EXP,
+				'EXP-MP-596-Aug2024', // Update with exp tracking ticket number
```

Comment:
Could you create an exp ticket under the same epic and put the ticket number here?